EasyRestProjectNetTeam2/EasyRestPages/ManageAdministratorPage.cs

```diff
@@ -0,0 +1,40 @@
+﻿using EasyRestProjectNetTeam2.Decorator;
+using EasyRestProjectNetTeam2.EasyRestComponentsObj;
+using OpenQA.Selenium;
+using SeleniumExtras.PageObjects;
+
+namespace EasyRestProjectNetTeam2.EasyRestPages
+{
+    public class ManageAdministratorPage : BasePage
+    {
+        public AddEmployeeComponent AddEmploeeComponent { get; private set; }
```

Comment:
please rename to AddEmplo**y**eeComponent

Reply:
Done.